vertical-pod-autoscaler/hack/vpa-process-yamls.sh

```diff
@@ -39,8 +39,17 @@ if [ $# -gt 2 ]; then
   exit 1
 fi
 
+SEMVER_RE='v[^0-9]*\([0-9]*\)[.]\([0-9]*\)[.]\([0-9]*\)\([0-9A-Za-z-]*\)'
+K8S_VERSION=$(kubectl version --short=true|grep Server|awk '{print $3}')
```

Comment:
I don't think we need to check the k8s version - the 0.9 VPA requires 1.16 anyway: https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler#compatibility